src/vs/editor/contrib/markdownRenderer/browser/markdownRenderer.ts

```diff
@@ -45,6 +46,7 @@ export class MarkdownRenderer {
 		private readonly _options: IMarkdownRendererOptions,
 		@ILanguageService private readonly _languageService: ILanguageService,
 		@IOpenerService private readonly _openerService: IOpenerService,
+		@IProductService private readonly _productService: IProductService,
```

Comment:
Looks like this won't work since `IProductService` doesn't exist for Monaco 

Will keep investigating to find another approach

Reply:
Might be easy to stub the `IProductService` for monaco. The interface looks huge but only a couple for fields are actually required: version and names